### PR TITLE
Add UEID type

### PR DESCRIPTION
**Pull request by setrofim**
Implement a type for representing UEID claims.

### CODE REVIEW

File: ueid_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+	u5 := UEID{
+		0x03, // EMEI
+		0xde, 0xad, 0xbe, 0xef, 0xde, 0xad, 0xbe, 0xef,
+		0xde, 0xad, 0xbe, 0xef, 0xde, 0xad, // 14 bytes
+	}
+	assert.Nil(u5.Validate())
+
+	u6 := UEID{
+		0x03, // EMEI
```
Comment (thomas-fossati) [inline ueid_test.go:50]: you need to be clement to my OCD: s/EMEI/IMEI/g :-)

Comment (setrofim) [reply ueid_test.go:50]: d'oh

**Review by thomas-fossati — COMMENTED** [review]
we'd need to also add UEID to the Eat map (in eat.go) and to the CBOR/JSON roundtrip tests (in eat_test.go).  Did you plan to do that in a separate PR?

**Comment by setrofim** [comment]
Ah, good point. I'll do that here.

**Review by thomas-fossati — APPROVED** [review]
🚢 it!

**Pull request merged**